src/main/kotlin/com/tschuchort/compiletesting/KaptComponentRegistrar.kt

```diff
@@ -96,7 +97,34 @@ internal class KaptComponentRegistrar(
         }
 
         AnalysisHandlerExtension.registerExtension(project, kapt3AnalysisCompletedHandlerExtension)
-        StorageComponentContainerContributor.registerExtension(project, Kapt3ComponentRegistrar.KaptComponentContributor())
+        StorageComponentContainerContributor.registerExtension(project,
```

Comment:
I have looked at the source code of the kapt plugin and it seems that we can just reuse `kapt3AnalysisCompletedHandlerExtension` here. Any particular reason why you declared a new `PartialAnalysisHandlerExtension`? I don't know if it will make a difference, since the tests still pass, but at the very least there won't be any logging by the `StorageComponentContainerContributor`.

See https://github.com/JetBrains/kotlin/blob/cf161f823471cc2cac33f39f398a4f67ee5b52b3/plugins/kapt3/kapt3-compiler/src/org/jetbrains/kotlin/kapt3/Kapt3Plugin.kt#L190

Reply:
I didn't see that class. I just stubbed a minimal no-op version of the class it expected

Reply:
1ac262c